js/app.js

```diff
@@ -0,0 +1,302 @@
+//
+// const isTheSidebarOpened = true;
+//
+// if (isTheSidebarOpened === true) {
+//     console.log('The sidebar is opened');
+// }
+//
+//
+// const myAge = 17;
+// const isAdult = myAge >= 18;
+//
+//
+//
+// if (isAdult) {
+//     console.log('It is legal for me to drink');
+// } else {
+//     console.log('It is not legal for me to drink');
+// }
+
+
+/*
+* Create two variables called width and height .
+* Each of those variables should hold some number.
+* Then, create a new variable that holds the area of a rectangle,
+*  calculated based on the width and height variables. Print it to the console.
+* */
+
+/*
+const value = 8 / 2; //
+// * mnożenie
+// / dzielenie
+// + dodawanie
+// - odejmowanie
+
+const width = 17;
+const height = 17;
+const rectangleArea = width * height;
+
+console.log(rectangleArea);
+
+/*
+* Create two variables called firstNumber and secondNumber.
+* Assign each of them with a different number. Write a program that swaps both values.
+* */
+
+/*
+let firstNumber = 12;
+let secondNumber = 14;
+
+const swapNumber = firstNumber;
+firstNumber = secondNumber;
+secondNumber = swapNumber;
+
+console.log('secondNumber', secondNumber);
+console.log('firstNumber', firstNumber);
+*/
+
+// 1 //
+let width = 10;
+let height = 10;
+
+let area = width*height;
+console.log('area', area);
+
+// 2 //
+let firstNumber= 11;
+let secondNumber= 22;
+
+const swapNumber = firstNumber; // if I use 'let' or 'const' i get the same result, are they both correct to use here?//
+firstNumber = secondNumber;
+secondNumber = swapNumber;
+
+console.log('secondNumber', secondNumber);
+console.log('firstNumber', firstNumber);
+
+// 3 //
+let physicsGrade = 3.8;
+let mathGrade = 4.5;
+let englishGrade = 4.7;
+
+let avarageGrade = (physicsGrade + mathGrade + englishGrade) / 3;
+const passedGrade = avarageGrade >= 4;
+console.log('Your avarage grade is:' + avarageGrade);
+
+if (passedGrade) {
+    console.log('Good Job!');
+} else {
+    console.log('Try harder!');
+}
+
+// 4 //
+let currentAge = 30;
+const retirenmentAgeInPoland = 67;
+let yearsLeftToRetirenment = retirenmentAgeInPoland;
+
+yearsLeftToRetirenment = retirenmentAgeInPoland - currentAge;
+
+console.log(yearsLeftToRetirenment);
+
+// 5 //
+const isPayingVAT = true;
+const VATRateInPoland = 0.23;
+const incomeTaxRate = 0.12;
+let monthlyIncome = 3500;
+let taxToPay = 0;
+
+if (isPayingVAT) {
+    taxToPay = monthlyIncome * VATRateInPoland;
+}
+
+taxToPay = taxToPay + monthlyIncome * incomeTaxRate;
+
+console.log('tax to pay: ' + taxToPay);
+
+// 6 //
+let isSunny = false;
+
+if (isSunny) {
+    console.log("don't forget your sunglasses!");
+} else {
+    console.log("you might need an umbrella.");
+}
+
+// 7 //
+let currentMonth = 9
+
+if (currentMonth <4 || currentMonth === 12) {
+    console.log("it's winter");
+} else {
+    console.log("it's not winter");
+}
+
+// 8 //
+let personWeight = 66
+let personHeight  = 1.74
+
+let BMI = personWeight / (personHeight * personHeight)
```

Comment:
```suggestion
let BMI = personWeight / (personHeight * personHeight);
```
Remember to add them everywhere where needed